### PR TITLE
Release 0.5.0

### PR DESCRIPTION
**Pull request by smaye81**
This release includes the following:

## Enhancements
* Make toJSON protected by @smaye81 in #335 
* Updated to Protobuf 21.12 and TypeScript 4.9.4 by @smaye81 in #338 and #341

### CODE REVIEW

**Comment by timostamm** [comment]
Can we merge https://github.com/bufbuild/protobuf-es/pull/307 before this, so that we release with the new readme on npmjs.com?

**Comment by smaye81** [comment]
> Can we merge #307 before this, so that we release with the new readme on npmjs.com?

Ah yep.  Good call.  I'll merge 307 in once it lands.

**Review by timostamm — APPROVED** [review]

**Pull request merged**
